### PR TITLE
reduce neutron agent check alerts

### PR DESCRIPTION
**Pull request by Xiaohua-Shen**
1. reduce keepalive threshold
2. only execute on controllers, not on computes

### CODE REVIEW

File: roles/common/templates/monitoring/config.json [hunk 1]
```diff
@@ -6,7 +6,11 @@
       "openstack"
     ],
     "keepalive": {
-      "handler": "{{ monitoring.keepalive.handler }}"
+      "handler": "{{ monitoring.keepalive.handler }}",
+	  "thresholds": {
```
Comment (paulczar) [inline roles/common/templates/monitoring/config.json]: if we're adding thresholds then we should make them configurable.  something like:

```
monitoring:
  keepalive:
    thresholds:
      warning: 40
      critical: 60
```

File: roles/neutron-data/tasks/monitoring.yml [hunk 2]
```diff
@@ -1,10 +1,4 @@
 ---
-- name: neutron agents check
-  sensu_check_dict:
-    name: "check-neutron-agents"
-    check: "{{ sensu_checks.neutron.check_neutron_agents }}"
-  notify: restart sensu-client missing ok
-
 - name: neutron agents duplicate check
```
Comment (paulczar) [inline roles/neutron-data/tasks/monitoring.yml]: shouldn't this task move with the other agent check ?

**Comment by Xiaohua-Shen** [comment]
@paulczar @scoop206 please review again.
1. add keepalive threshold as configurable parameter
2. move all neutron-data related monitoring to neutron-control, since they are controller level check. Although operation team only mentioned check-neutron-agent alert flooding.

**Comment by Xiaohua-Shen** [comment]
retest

File: roles/neutron-data/tasks/main.yml [hunk 3]
```diff
@@ -56,8 +56,3 @@
             dest=/etc/cron.daily/cleanup-neutron-interfaces
             mode=0755 owner=root
 
-- include: monitoring.yml
```
Comment (paulczar) [inline roles/neutron-data/tasks/main.yml]: we should keep this and just have the monitoring.yml task be empty.

File: roles/neutron-data/tasks/monitoring.yml [hunk 4]
```diff
@@ -1,22 +0,0 @@
----
```
Comment (paulczar) [inline roles/neutron-data/tasks/monitoring.yml]: keep this file,  just be empty apart from the `---` line.

**Comment by paulczar** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Comment by Xiaohua-Shen** [comment]
retest

**Pull request merged**